### ISSUE TITLE
Bundler LoadError on jekyll serve

### ISSUE THREAD

**Issue by inspiredtolive** [jekyll/jekyll#5165]
- [ ] I believe this to be a bug, not a question about using Jekyll.
- [ ] I Updated to the latest Jekyll (or) if on Github Pages to the latest `github-pages`
- [ ] I Read the CONTRIBUTION file at https://jekyllrb.com/docs/contributing/
- [ ] This is a feature request.

---

- [x] I am on (or have tested on) ***macOS*** 10+
- [ ] I am on (or have tested on) ***Debian/Ubuntu*** GNU/Linux
- [ ] I am on (or have tested on) ***Fedora*** GNU/Linux
- [ ] I am on (or have tested on) ***Arch*** GNU/Linux
- [ ] I am on (or have tested on) ***Other*** GNU/Linux
- [ ] I am on (or have tested on) ***Windows*** 10+

---

- [x] I was trying to install.
- [ ] There is a broken Plugin API.
- [ ] I had an error on Github Pages, and I have not tested locally.
- [ ] I had an error on Github Pages, and Github Support said it was a Jekyll Bug.
- [ ] I had an error on Github Pages and I did not test it locally.
- [ ] I was trying to build.
- [ ] It was another bug.

## My Reproduction Steps

I tried to follow the quick-start guide.

```
~ $ gem install jekyll
~ $ jekyll new .
~ $ jekyll serve
```

## The Output I Wanted

I wanted to run jekyll serve to preview the generated site, but I got this.

```
/Library/Ruby/Site/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- bundler (LoadError)
	from /Library/Ruby/Site/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /Library/Ruby/Gems/2.0.0/gems/jekyll-3.2.0/lib/jekyll/plugin_manager.rb:34:in `require_from_bundler'
	from /Library/Ruby/Gems/2.0.0/gems/jekyll-3.2.0/exe/jekyll:9:in `<top (required)>'
	from /usr/local/bin/jekyll:23:in `load'
	from /usr/local/bin/jekyll:23:in `<main>'
```

## Additional comments
I already installed:

- Ruby (via brew install ruby)
- RubyGems (via gem update --system)

**Comment by parkr** [comment]
Try running:

```text
$ gem install bundler
$ bundle install
$ bundle exec jekyll serve
```

We may need to update our "getting started" instructions when using `jekyll new`!

**Comment by inspiredtolive** [comment]
I am getting this when I run gem install bundler:

```
ERROR:  While executing gem ... (Errno::EPERM)
    Operation not permitted - /usr/bin/bundle
```

**Comment by parkr** [comment]
@inspiredtolive Interesting! It looks like you're using the system Ruby version (pre-installed) and it may have some issues with permissions. You can either install into `~/bin`, or use another installation of Ruby, such as the one provided by Homebrew: `brew install ruby`. Would either of those solutions work for you?

**Comment by inspiredtolive** [comment]
**SOLVED**

TLDR: I already had ruby via Homebrew, but it was not linked. I believe it was because I had rake installed and it did not want to overwrite it. I let brew link ruby using the --override option and it let me run gem install bundler.

**What I Did To Solve My Problem**

I ran brew install ruby and I got this:

```
brew install ruby
Warning: ruby-2.3.1 already installed, it's just not linked
```

I ran brew link --overwrite --dry-run ruby and I got this:

```
Would remove:
/usr/local/bin/rake
/usr/local/share/emacs/site-lisp/autoconf/autoconf-mode.el
/usr/local/share/emacs/site-lisp/autoconf/autoconf-mode.elc
/usr/local/share/emacs/site-lisp/autoconf/autotest-mode.el
/usr/local/share/emacs/site-lisp/autoconf/autotest-mode.elc
```

so I ran brew link --overwrite ruby and I got this

`Linking /usr/local/Cellar/ruby/2.3.1... 1230 symlinks created`

And I was able to run the following commands successfully:

```
$ gem install bundler
$ bundle install
$ bundle exec jekyll serve
```

**Comment by farskid** [comment]
@parkr Instructions solved my issue on mac os

**Comment by zfc580** [comment]
I have brew link --overwrite --dry-run ruby as @inspiredtolive said, but still contain the error:`ERROR:  While executing gem ... (Gem::FilePermissionError)
    You don't have write permissions for the /Library/Ruby/Gems/2.0.0 directory.`what steps did miss?

**Comment by codedentwickler** [comment]
@zfc580 you need to add sudo to install, so you have the required permission.
$ sudo gem install bundler
$ bundle install
$ bundle exec jekyll serve

**Comment by osiyuk** [comment]
Interesting, I have almost the same issue with jekyll. In my case I am using debian/linux system.

**Comment by 94mkj** [comment]
> Try running:
> 
> ```
> $ gem install bundler
> $ bundle install
> $ bundle exec jekyll serve
> ```
> 
> We may need to update our "getting started" instructions when using `jekyll new`!

**Comment by kashura** [comment]
This helped a ton. Thank you.
